### PR TITLE
Docs: Refactor inconsistent unordered lists

### LINKED ISSUE #27818: Update markdown bullet format
**What would you like to be added**:

Change all unordered lists in the Grafana technical documentation from `*` to *`-`.

Example:
```
* One
* Two
```

Becomes:
```
- One
- Two
```

**Why is this needed**:

For consistency and to avoid confusion when the first word of the list item is formatted bold.

### PR DESCRIPTION
**Pull request by outofgamut**
Fixes #27818

### CODE REVIEW

**Comment by oddlittlebird** [comment]
Please check the conflicting files. We just moved some content out of the features folder and into new locations in the directory.

**Comment by outofgamut** [comment]
@oddlittlebird - conflicts should be resolved and rebased on master

File: contribute/style-guides/documentation-markdown-guide.md [hunk 1]
```diff
@@ -12,34 +12,35 @@ HTML heading levels:
...
-* Make text **bold** using two asterisks.
+- Make text **bold** using two asterisks.
 
 **Example:** It is ``**important**`` to use GitHub Flavored Markdown emoji consistently.
 
-* Make text ``_emphasized_`` using single `` _underscores_``. Do not use the single asterisk, it can be easily confused with bold.
+- Make text ``*emphasized*`` using single `` _underscores_`` or a single asterisk.
 
-**Example:** GitHub-flavored markdown emoji should _only_ appear in specific cases.
+**Example:** GitHub Flavored Markdown emoji should _only_ appear in specific cases.
```
Comment (oddlittlebird) [inline contribute/style-guides/documentation-markdown-guide.md:29]: Please revert this change.

File: contribute/style-guides/documentation-markdown-guide.md [hunk 2]
```diff
@@ -12,34 +12,35 @@ HTML heading levels:
...
 
 ## Bold and emphasis
 
-* Make text **bold** using two asterisks.
+- Make text **bold** using two asterisks.
 
 **Example:** It is ``**important**`` to use GitHub Flavored Markdown emoji consistently.
 
-* Make text ``_emphasized_`` using single `` _underscores_``. Do not use the single asterisk, it can be easily confused with bold.
+- Make text ``*emphasized*`` using single `` _underscores_`` or a single asterisk.
```
Comment (oddlittlebird) [inline contribute/style-guides/documentation-markdown-guide.md:27]: Please revert this change. The other sentence is correct.

File: contribute/style-guides/documentation-markdown-guide.md [hunk 3]
```diff
@@ -12,34 +12,35 @@ HTML heading levels:
...
 
-**Example:** GitHub-flavored markdown emoji should _only_ appear in specific cases.
+**Example:** GitHub Flavored Markdown emoji should _only_ appear in specific cases.
 
 
 ## Links and references
 
-Create links to other website by wrapping the display text in square brackets, and the web URL in curved brackets.
+Create links to other website by wrapping the display text in square brackets, and
```
Comment (oddlittlebird) [inline contribute/style-guides/documentation-markdown-guide.md:34]: Please do not make weird line breaks. It's easier to edit sentences when they're not broken into multiple lines.

File: contribute/style-guides/documentation-markdown-guide.md [hunk 4]
```diff
@@ -12,34 +12,35 @@ HTML heading levels:
...
+the web URL in curved brackets.
 
 \[text to display](www.website.com)
 
 **Example:** For more information on including emoji in GitHub flavored Markdown, refer to the [webfx page on emoji](https://www.webfx.com/tools/emoji-cheat-sheet/) for a list of emoji.
 
 ## Block quotes
 
-Include block quotes inside text using right-facing arrows:
+Include Block quotes inside text using right-facing arrows:
```
Comment (oddlittlebird) [inline contribute/style-guides/documentation-markdown-guide.md:43]: Please revert this change.

File: contribute/style-guides/documentation-markdown-guide.md [hunk 5]
```diff
@@ -116,7 +117,7 @@ The list above will always display as:
 ### Unordered lists
 
 Build a list of points - an unordered or unnumbered list - by
-using "\-" (hyphen) characters.
+using "\-" characters.
```
Comment (oddlittlebird) [inline contribute/style-guides/documentation-markdown-guide.md:120]: Please revert this change.

File: contribute/style-guides/documentation-markdown-guide.md [hunk 6]
```diff
@@ -134,8 +135,10 @@ This follows the format of "!", alt text wrapped in "[]" and the link URL wrappe
 
 ## Comments
 
-You can include comments that will not appear in published markdown using the following syntax:
+You can include comments that will not appear in published markdown using the
```
Comment (oddlittlebird) [inline contribute/style-guides/documentation-markdown-guide.md:138]: See earlier comment about adding weird line breaks. Apply throughout.

File: docs/sources/auth/auth-proxy.md [hunk 7]
```diff
@@ -108,25 +108,25 @@ In this example we use Apache as a reverse proxy in front of Grafana. Apache han
     </VirtualHost>
 ```
 
-* The first 4 lines of the virtualhost configuration are standard, so we won’t go into detail      on what they do.
+- The first 4 lines of the virtualhost configuration are standard, so we won’t go into detail      on what they do.
```
Comment (oddlittlebird) [inline docs/sources/auth/auth-proxy.md:111]:
```suggestion
- The first four lines of the virtualhost configuration are standard, so we won’t go into detail on what they do.
```

File: docs/sources/auth/auth-proxy.md [hunk 8]
```diff
@@ -108,25 +108,25 @@ In this example we use Apache as a reverse proxy in front of Grafana. Apache han
...
+        - **RequestHeader set X-WEBAUTH-USER “%{PROXY_USER}e”**: With the authenticated username now stored in the PROXY_USER variable, we create a new HTTP request header that will be sent to our backend Grafana containing the username.
 
-* The **RequestHeader unset Authorization** removes the Authorization header from the HTTP request before it is forwarded to Grafana. This ensures that Grafana does not try to authenticate the user using these credentials (BasicAuth is a supported authentication handler in Grafana).
+- The **RequestHeader unset Authorization** removes the Authorization header from the HTTP request before it is forwarded to Grafana. This ensures that Grafana does not try to authenticate the user using these credentials (BasicAuth is a supported authentication handler in Grafana).
 
-* The last 3 lines are then just standard reverse proxy configuration to direct all authenticated requests to our Grafana server running on port 3000.
+- The last 3 lines are then just standard reverse proxy configuration to direct all authenticated requests to our Grafana server running on port 3000.
 
 ## Full walk through using Docker.
```
Comment (oddlittlebird) [inline docs/sources/auth/auth-proxy.md:125]:
```suggestion
## Full walkthrough using Docker.
```

File: docs/sources/auth/auth-proxy.md [hunk 9]
```diff
@@ -108,25 +108,25 @@ In this example we use Apache as a reverse proxy in front of Grafana. Apache han
...
 
 For this example, we use the official Grafana docker image available at [Docker Hub](https://hub.docker.com/r/grafana/grafana/)
```
Comment (oddlittlebird) [inline docs/sources/auth/auth-proxy.md:127]:
```suggestion
For this example, we use the official Grafana Docker image available at [Docker Hub](https://hub.docker.com/r/grafana/grafana/).
```

File: docs/sources/auth/auth-proxy.md [hunk 10]
```diff
@@ -207,13 +207,13 @@ ProxyPass / http://grafana:3000/
...
 
-* Create a htpasswd file. We create a new user **anthony** with the password **password**
+- Create a htpasswd file. We create a new user **anthony** with the password **password**
 
     ```bash
     htpasswd -bc htpasswd anthony password
     ```
 
-* Launch the httpd container using our custom httpd.conf and our htpasswd file. The container will listen on port 80, and we create a link to the **grafana** container so that this container can resolve the hostname **grafana** to the grafana container’s ip address.
+- Launch the httpd container using our custom httpd.conf and our htpasswd file. The container will listen on port 80, and we create a link to the **grafana** container so that this container can resolve the hostname **grafana** to the grafana container’s ip address.
```
Comment (oddlittlebird) [inline docs/sources/auth/auth-proxy.md:216]:
```suggestion
- Launch the httpd container using our custom httpd.conf and our htpasswd file. The container will listen on port 80, and we create a link to the **grafana** container so that this container can resolve the hostname **grafana** to the Grafana container’s IP address.
```

File: docs/sources/guides/whats-new-in-v6-0.md [hunk 11]
```diff
@@ -143,8 +143,8 @@ Read more about the short-lived token solution and how to configure it [here](/a
 
 Besides these changes we have also made security improvements regarding Cross-Site Request Forgery (CSRF) and Cross-site Scripting (XSS) vulnerabilities:
 
-* Cookies are per default using the [SameSite](/administration/configuration/#cookie-samesite) attribute to protect against CSRF attacks
-* Script tags in text panels are per default [disabled](/administration/configuration/#disable-sanitize-html) to protect against XSS attacks
+- Cookies are per default using the [SameSite](/administration/configuration/#cookie-samesite) attribute to protect against CSRF attacks
+- Script tags in text panels are per default [disabled](/administration/configuration/#disable-sanitize-html) to protect against XSS attacks
 
 > If you're using [Auth Proxy Authentication](/auth/auth-proxy/) you still need to have user sessions set up and configured
```
Comment (oddlittlebird) [inline docs/sources/guides/whats-new-in-v6-0.md:149]:
```suggestion
> **Note:** If you're using [Auth Proxy Authentication](/auth/auth-proxy/) you still need to have user sessions set up and configured
```

File: docs/sources/http_api/_index.md [hunk 12]
```diff
@@ -19,28 +19,28 @@ dashboards, creating users and updating data sources.
 ## Supported HTTP APIs:
```
Comment (oddlittlebird) [inline docs/sources/http_api/_index.md:19]:
```suggestion
## Supported HTTP APIs
```

File: docs/sources/linking/data-link-variables.md [hunk 13]
```diff
@@ -20,31 +20,31 @@ You can also use template variables in your data links URLs, refer to [Templates
...
+- `$__from and $__to` - For more information, refer to [Global variables]({{< relref "../variables/variable-types/global-variables.md#__from-and-__to" >}}).
 
 ## Series variables
 
 Series specific variables are available under ``__series`` namespace:
 
-* ``__series.name`` - series name to the URL
-* ``__series.labels.<LABEL>`` - label's value to the URL. If your label contains dots use ``__series.labels["<LABEL>"]`` syntax
+- ``__series.name`` - series name to the URL
+- ``__series.labels.<LABEL>`` - label's value to the URL. If your label contains dots use ``__series.labels["<LABEL>"]`` syntax
```
Comment (oddlittlebird) [inline docs/sources/linking/data-link-variables.md:31]:
```suggestion
- ``__series.labels.<LABEL>`` - label's value to the URL. If your label contains dots, then use ``__series.labels["<LABEL>"]`` syntax.
```

File: docs/sources/reference/share_panel.md [hunk 14]
```diff
@@ -13,7 +13,7 @@ Click a panel title to open the panel menu, then click share in the panel menu t
 
 ## Direct Link Rendered Image
 
-You also get a link to render a .png image of the panel, which is useful if you want to share an image of the panel. For more information about the requirements and how to configure image rendering, refer to [Image rendering]({{< relref "../administration/image_rendering.md)" >}}.
+You also get a link to render a PNG image of the panel. Useful if you want to share an image of the panel. Read more about the requirements and how to configure image rendering [here](/administration/image_rendering/).
```
Comment (oddlittlebird) [inline docs/sources/reference/share_panel.md:16]: Please revert this change.

**Review by oddlittlebird — CHANGES_REQUESTED** [review]
Wow, thank you so much! Once you address my few edits, we'll get this merged.

**Review by hugohaggmark — COMMENTED** [review]
I'm going to exclude myself from this PR as I can see that there are others that are engaged.

**Review by oddlittlebird — APPROVED** [review]
Excellent. Thanks for contributing to the docs, and Happy Hacktoberfest!

**Comment by outofgamut** [comment]
No problem @oddlittlebird ! Was a fun exercise in using regex patterns for find-and-replace in vscode 😄  (and good warmup for Hacktoberfest). Thanks for working through it with me!

**Comment by oddlittlebird** [comment]
Please address the conflicting file, then I can get this merged.

**Comment by outofgamut** [comment]
conflict addressed. should be good to merge.
